mopidy_spotify/playlists.py

```diff
@@ -47,7 +54,15 @@ def _get_playlist(self, uri, as_items=False):
         )
 
     def refresh(self):
-        pass  # TODO: Clear/invalidate all caches on refresh.
+        with utils.time_logger("Refresh Playlists", logging.INFO):
+            _cache.clear()
+            count = 0
+            for playlist_ref in self._get_flattened_playlist_refs():
+                self._get_playlist(playlist_ref.uri)
+                count = count + 1
+            logger.info(f"Refreshed {count} playlists")
```

Comment:
INFO level messages should have enough context to make sense without the logger name, so replace "playlists" with "Spotify playlists".